TUFTOOL_EXAMPLES.md

```diff
@@ -0,0 +1,156 @@
+# Tuftool Examples
+
+## Creating and Adding New Roles
+
+The following example creates 2 roles, role1 and role2. role1 is a delegated targets of targets and role 2 is a delegated targets of role1.
+
+### Create a new role named role1
+
+`tuftool delegation --signing-role role1 create \`\
+`--key path/to/role1/key \`\
+`--expires in 2 days \`\
+`--version 1 \`\
+`--outdir role1/destination`\
```

Comment:
Let's wrap these examples in a triple backtick, so each line doesn't show up separately when it is rendered.

```
tuftool foo \
    --bar ...\
    --baz ...
```

This looks a little funny :)

`tuftool delegation --signing-role role1 create \`\
`--key path/to/role1/key \`\
`--expires in 2 days \`\
`--version 1 \`\